### PR TITLE
feat: colors enum bb-23

### PR DESCRIPTION
**Pull request by Shcracoziabra**
**Purpose**: creating enum for app colors.
**Overview**: added enums for base and gradient colors.

### CODE REVIEW

File: apps/mobile/src/libs/enums/app/common-colors.enum.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+const CommonColors = {
```
Comment (fshabanov) [inline apps/mobile/src/libs/enums/app/common-colors.enum.ts:1]: B23

File: package-lock.json
Comment (fshabanov) [inline package-lock.json:1]: There shouldn't be a change in this file if no library was installed/removed

**Review by fshabanov — CHANGES_REQUESTED** [review]

File: apps/mobile/src/libs/enums/app/common-colors.enum.ts (hunk 1 above)
Comment (anogire) [reply apps/mobile/src/libs/enums/app/common-colors.enum.ts:1]: I think we can use just "Color"

File: apps/mobile/src/libs/enums/app/common-colors.enum.ts
Comment (revenets) [inline apps/mobile/src/libs/enums/app/common-colors.enum.ts:1]: Let's create two separate enums for base colors and gradient ones. And change gradient values to arrays of two colors, for example: `BLUE_BTN_GT: ['#00F0FF', '#0085FF']`, Also, let's stick with one color format rgba or hex (I propose to keep all the colors in rgba to be more flexible in the transparency adjustment)

**Review by revenets — CHANGES_REQUESTED** [review]

File: apps/mobile/src/libs/enums/app/common-colors.enum.ts
Comment (Shcracoziabra) [reply apps/mobile/src/libs/enums/app/common-colors.enum.ts:1]: What about degrees and percents used in gradients? Is there no need to put them in the enum?

Comment (revenets) [reply apps/mobile/src/libs/enums/app/common-colors.enum.ts:1]: No, in the further implementation, these params will be set separately for the <LinearGradient />. 
You can look [here](https://www.npmjs.com/package/react-native-linear-gradient), how it works

File: apps/mobile/src/libs/components/input/styles.ts [hunk 2]
```diff
@@ -1,8 +1,11 @@
 import { StyleSheet } from "react-native";
 
+import { GradientColor } from "~/libs/enums/enums";
+
+const colorIndex = 0;
 const styles = StyleSheet.create({
 	input: {
-		borderColor: "gray",
+		borderColor: GradientColor.BLUE[colorIndex],
```
Comment (revenets) [inline apps/mobile/src/libs/components/input/styles.ts:8]: In this case we can do just like this
```suggestion
		borderColor: GradientColor.BLUE[0],
```

Comment (Shcracoziabra) [reply apps/mobile/src/libs/components/input/styles.ts:8]: Linter complains "no magic numbers" in this case, shoud I comment linter rule above?

Comment (revenets) [reply apps/mobile/src/libs/components/input/styles.ts:8]: OK, leave the `colorIndex` then.

File: apps/mobile/src/libs/components/input/styles.ts [hunk 3]
```diff
@@ -1,8 +1,11 @@
 import { StyleSheet } from "react-native";
 
+import { GradientColor } from "~/libs/enums/enums";
+
+const colorIndex = 0;
```
Comment (fshabanov) [inline apps/mobile/src/libs/components/input/styles.ts:5]: B22

**Review by fshabanov — CHANGES_REQUESTED** [review]

File: apps/mobile/src/libs/components/input/styles.ts (hunk 3 above)
Comment (fshabanov) [reply apps/mobile/src/libs/components/input/styles.ts:5]: Not fixed

**Review by fshabanov — CHANGES_REQUESTED** [review]

File: apps/mobile/src/libs/components/input/styles.ts (hunk 3 above)
Comment (Shcracoziabra) [reply apps/mobile/src/libs/components/input/styles.ts]: My bad, now corrected it.

**Review by fshabanov — APPROVED** [review]

**Review by revenets — APPROVED** [review]

**Pull request merged**